app/helpers/hyrax/hyrax_helper_behavior.rb

```diff
@@ -54,6 +54,23 @@ def browser_supports_directory_upload?
       user_agent.include? 'Chrome'
     end
 
+    # @param [User] user
+    def render_notifications(user:)
+      mailbox = UserMailbox.new(user)
+      unread_notifications = mailbox.unread_count
+      link_to(hyrax.notifications_path,
+              'aria-label' => mailbox.label(params[:locale]),
+              class: 'notify-number') do
+        capture do
+          concat content_tag(:span, '', class: 'fa fa-bell')
+          concat "\n"
```

Comment:
It looks janky but the newline here preserves the prior appearance as set by the `notify_number` partial.

Reply:
lol - but looking better to the user is the important thing